scripts/bootstrap.sh

```diff
@@ -119,15 +119,18 @@ if [ -d /vagrant ]; then
     # by default the project's top.sls is disabled by file naming. hook that up here
     cd /srv/salt/ && ln -sf example.top top.sls
     
-    # install the builder base formula 
-    if [ ! -d /vagrant/cloned-projects/builder-base-formula/.git ]; then
-        git clone https://github.com/elifesciences/builder-base-formula \
-            /vagrant/cloned-projects/builder-base-formula
-    fi
-    
-    # overwrite the general purpose /etc/salt/minion file created above with 
-    # this more complex one for dev environments only
-    cp /vagrant/scripts/salt/minion /etc/salt/minion
+    # vagrant makes all formula dependencies available, including builder base formula
+
+    # overwrite the general purpose /etc/salt/minion file created above 
+    # with this more complex one for dev environments only
+    cp /vagrant/scripts/salt/minion.template /etc/salt/minion
```

Comment:
looking back at this now, here we are in a long conditional that is only executed in the Vagrant case. Do we need the else at all? The original `/vagrant/scripts/salt/minion.template` shouldn't work at all as it's missing the `builder-base-formula`; therefore I propose to put an `exit 1` inside the else here rather than using it and finding out some states are missing.
This would become something like:
```
    custom_minion_file="/vagrant/scripts/salt/$minion_id.minion"
    if [ ! -e "$custom_minion_file" ]; then
        echo "couldn't find $custom_minion_file"
        exit 1
    fi
    cp "$custom_minion_file" /etc/salt/minion
```